.github/workflows/build-image.yml

```diff
@@ -0,0 +1,26 @@
+name: Docker registry
+
+on: 
+   push:
+     tags:
+       - 'v*'
+
+jobs:
+  github:
```

Comment:
better name for the job would be sth. like `push-dockerhub` (than `github`)

Reply:
I changed it to `push_dockerhub`